src/modules/jobPost/jobPost.controller.ts

```diff
@@ -16,6 +16,16 @@ export class JobPostController {
     return this.jobPostService.getJobPostByUser(Number(userId));
   }
 
+  @Get()
+  getAllJobPost() {
+    return this.jobPostService.getAllJobPost();
+  }
+
+  @Get('skill/:id')
+  findJobPostByUserId(@Param('id') userId: number) {
```

Comment:
return types?

Reply:
I do this to return the job posts by userId with the user skills and category. But I think I can use another function which have the path user/:id.